cmd/goal/README.md

```diff
@@ -125,4 +125,17 @@ goal app call --from ${ACCOUNT} --app-id ${APPID} --box "str:an_ABI_box" --app-a
 
 # set the contents to ABI type `(uint8,uint8,uint8)` with value `[2,3,5]`
 goal app call --from ${ACCOUNT} --app-id ${APPID} --box "$str:an_ABI_box" --app-arg "str:set" --app-arg "str:an_ABI_box"  --app-arg "abi:(uint8,uint8,uint8):[2,3,5]"
-```
\ No newline at end of file
+```
+
+### Q: How do I search for boxes in goal?
+
+### A:
+Assuming you followed the previous step to create `greatBox` and `an_ABI_box`:
+
+```sh
+# get all boxes for a given app
+goal app box list --app-id ${APPID}
+
+# get the box details for a given box
+goal app box info --app-id ${APPID} --box "str:an_ABI_box"
```

Comment:
```suggestion
goal app box info --app-id ${APPID} --name "str:an_ABI_box"
```

Almost [WOMM](https://www.urbandictionary.com/define.php?term=WOMM), except I had to make this change.